src/shared/lib/AlterationsUtils.ts

```diff
@@ -0,0 +1,78 @@
+export type FilteredOutAlterations = {
+    /**
+     * how many alterations are not shown
+     */
+    notShownAlteredCases: number;
+    /**
+     * number of genes associated with the hidden alterations
+     */
+    affectedGenes: number;
+};
+
+/**
+ * Returns true if the left map contains all genes from the right map with the same number of correspondent alterations.
+ * @param leftAlteredCasesPerGene superset candidate
+ * @param rightAteredCasesPerGene subset candidate
+ */
+export function subset(
```

Comment:
I have remove this code since it was unused. It was related to banners showing in study view and comparison view showing number of filtered out genes. There had been a decision to drop this and I forgot to take out the code that Ruslan added for this.